PhysicsTools/PatAlgos/plugins/PATPhotonProducer.cc

```diff
@@ -370,6 +372,38 @@ void PATPhotonProducer::produce(edm::Event & iEvent, const edm::EventSetup & iSe
     aPhoton.setCryEta( ecalRegData.seedCrysEtaOrX() );
     aPhoton.setIEta( ecalRegData.seedCrysIEtaOrIX() );
     aPhoton.setIPhi( ecalRegData.seedCrysIPhiOrIY() );
+    } else {
+    aPhoton.setEMax(0);
```

Comment:
I have kept the possibility of not filling the regression data in the producer as I think it can be useful in general: there may be PAT workflows other than miniAOD for which the slimmer is not run but that data is not useful either.